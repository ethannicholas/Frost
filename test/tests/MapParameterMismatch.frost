method main() {
    -- enumeration returns a tuple, so the map should only take one parameter, not two. It might
    -- make sense to extend this to actually work in the future, but for right now we make sure we
    -- get a sensible error out of this.
    Console.printLine(["Hello", "Goodbye"].enumeration.map((index, string) => "\{index}: \{string}"))
}