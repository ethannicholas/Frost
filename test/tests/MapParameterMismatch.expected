MapParameterMismatch.frost:5:60: error: expected '((frost.core.Int, frost.core.String))=&>(frost.collections.Iterator.map.U)', but found lambda with wrong number of parameters
1 error
